### ISSUE TITLE
False positive on NestedFunctionCalls check

### ISSUE THREAD

**Issue by aptinio** [rrrene/credo#1021]
### Environment

* Credo version (`mix credo -v`): 1.7.0-dev-ref.master.1c1b99ea
* Erlang/Elixir version (`elixir -v`): Elixir 1.14.2 (compiled with Erlang/OTP 25)
* Operating system: Linux

### What were you trying to do?

```elixir
opts |> Keyword.get(String.to_existing_atom(key), key) |> to_string()
```

### Expected outcome

No issue raised.

### Actual outcome

"Use a pipeline when there are nested function calls" suggested.

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by aptinio** [comment]
I'm unable to reproduce this on 1.7.0.
